api/capi/src/nnstreamer-capi-single-new.c

```diff
@@ -422,13 +422,16 @@ ml_single_open (ml_single_h * single, const char *model,
 
   if (!available) {
     ml_loge ("The given nnfw is not available.");
-    status = ML_ERROR_NOT_SUPPORTED;
-    return status;
+    return ML_ERROR_NOT_SUPPORTED;
   }
 
   /** Create ml_single object */
   single_h = g_new0 (ml_single, 1);
-  g_assert (single_h);
+  if (single_h == NULL) {
+    ml_loge ("Failed to allocate the single handle.");
+    return ML_ERROR_STREAMS_PIPE;
```

Comment:
ML_ERROR_STREAM_PIPE is described as `/**< Cannot create or access GStreamer pipeline. */`
As the single-new does not use GStreamer, how about some other error?
Is it possible to introduce equivalent of ENOMEM?

Reply:
I will change this to ML_ERROR_UNKNOWN now.
But later, we need to update this error code.

TIZEN_ERROR_OUT_OF_MEMORY = -ENOMEM
I will mark TODO to replace no-mem error code.
Let's update error code after 5.5 M2.